src/components/search/Search.svelte

```diff
@@ -62,15 +85,14 @@ async function onKeypress(event) {
     align-items: stretch;
     background-color: var(--input-background-color);
     border-radius: var(--space-1h);
-
+    position: relative;
```

Comment:
Is this needed? I don't see a difference but I may be missing something.

Reply:
It was with the old error notice but not anymore. Good find.